backend/app/schemas.py

```
@@ -213,9 +206,18 @@ class OrderDetailOut(BaseModel):
 
+
+class PayBillRequest(BaseModel):
+    order_id: int
+    method: str
+    trx_id: str
+
+
 class ChatRequest(BaseModel):
-    message: str = Field(..., min_length=1, max_length=1000) 
-    
+    messages: List[str]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)